editor.planx.uk/src/pages/FlowEditor/lib/store/navigation.ts

```diff
@@ -133,7 +133,7 @@ export const navigationStore: StateCreator<
 
     const sectionStatuses: Record<string, SectionStatus> = {};
     Object.keys(sectionNodes).forEach((sectionId) => {
-      if (updatedSectionNodeIds?.includes(sectionId)) {
+      if (updatedNodeIds?.includes(sectionId)) {
```

Comment:
this is just a minor argument naming update that I failed to properly commit/propagate through from previous work here https://github.com/theopensystemslab/planx-new/pull/1535#discussion_r1138668346)